modules/tinymce/src/core/test/ts/browser/keyboard/ArrowKeysInlineBoundariesTest.ts

```diff
@@ -336,4 +336,27 @@ describe('browser.tinymce.core.keyboard.ArrowKeysInlineBoundariesTest', () => {
       TinyAssertions.assertCursor(editor, [ 0, 0 ], 0);
     });
   });
+
+  context('Arrow keys on inline boundary node when selection is not collapsed', () => {
+    it('TINY-8601: It should move caret before selected inline boundary node when arrow left', () => {
```

Comment:
We shouldn't start this with `it`, so maybe something like this?
```suggestion
    it('TINY-8601: should move caret before selected inline boundary node when arrow left', () => {
```